src/pip/_internal/build_env.py

```diff
@@ -70,17 +70,19 @@ def __init__(self):
         # - ensure .pth files are honored
         # - prevent access to system site packages
         system_sites = {
-            os.path.normcase(site) for site in (
+            os.path.normcase(site)
+            for site in (
                 get_python_lib(plat_specific=False),
                 get_python_lib(plat_specific=True),
             )
         }
-        self._site_dir = os.path.join(temp_dir.path, 'site')
+        self._site_dir = os.path.join(temp_dir.path, "site")
         if not os.path.exists(self._site_dir):
             os.mkdir(self._site_dir)
-        with open(os.path.join(self._site_dir, 'sitecustomize.py'), 'w') as fp:
-            fp.write(textwrap.dedent(
-                '''
+        with open(os.path.join(self._site_dir, "sitecustomize.py"), "w") as fp:
+            fp.write(
+                textwrap.dedent(
+                    """
                 import os, site, sys
```

Comment:
We probably need to reindent the code literal here.

Also, could we please have 79-column limit?

![](https://static.tvtropes.org/pmwiki/pub/images/7582720_4794.jpg)